src/Server.h

```diff
@@ -144,6 +152,9 @@ class cServer
 	from the settings. */
 	bool ShouldAllowMultiWorldTabCompletion(void) const { return m_ShouldAllowMultiWorldTabCompletion; }
 
+	/** Get the Forge mods registered for a given protocol. */
+	AStringMap & GetRegisteredForgeMods(const UInt32 a_Protocol);
```

Comment:
They could but then the body of GetRegisteredForgeMods() would need to be reimplemented. Suppose I could make it private then have a public accessor which is const.